documentation/book/assembly-upgrade.adoc

```diff
@@ -3,20 +3,83 @@
 // master.adoc
 
 [id='assembly-upgrade-{context}']
-= {ProductName} and Kafka upgrades
+= Upgrade
 
-Each version of {ProductName} supports a range of versions of Apache Kafka.
-You can upgrade to a higher Kafka version as long as that version is supported by your version of {ProductName}.
+You can upgrade {ProductName} with no cluster downtime. Each version of {ProductName} supports a range of versions of Apache Kafka. You can upgrade to a higher Kafka version as long as that version is supported by your version of {ProductName}.
 In some cases, you can also downgrade to a lower supported Kafka version.
 
-When a newer version of {ProductName} is available, it may provide support for newer versions of Kafka. 
-Therefore, you will need to upgrade to the new version of {ProductName} before you can upgrade to a higher supported Kafka version.
-Upgrading the version of {ProductName} is done by upgrading the Cluster Operator deployment to the new version.
+Later versions of {ProductName} provide support for newer versions of Kafka, so you need to upgrade {ProductName} _before_ you can upgrade to a higher supported Kafka version.
 
 ifdef::StrimziUpgrades[]
-include::proc-upgrading-the-cluster-operator-0-10-0-to-0-11-0.adoc[leveloffset=+1]
+*Kafka versions*
+
+The Kafka message format version and inter-broker protocol version are subject to change. As a result, the upgrade process involves making both configuration changes to existing Kafka brokers and code changes to client applications (consumers and producers). The following table shows the differences between the Kafka versions:
+
+include::snip-kafka-versions.adoc[leveloffset=+1]
+
+endif::StrimziUpgrades[]
+
+ifndef::StrimziUpgrades[]
+*Kafka versions*
+
+{ProductName} 1.0.0 and 1.1.0 are based on different versions of Apache Kafka:
+
+[options="header"]
+|=======================
+|{ProductName} version |Kafka version
+|1.0.0                 |2.0.0
+|1.1.0                 |2.1.1
+|=======================
+
+The Kafka message format version and inter-broker protocol version are subject to change. As a result, the upgrade process involves making both configuration changes to existing Kafka brokers and code changes to client applications (consumers and producers). The following table shows the differences between the Kafka versions:
+
+[options="header"]
+|=======================
+|Kafka version |Interbroker protocol version  |Log message format version | Zookeeper version
+|2.0.0         |2.0                           |2.0                        | 3.4.13
+|2.1.1         |2.1                           |2.1                        | 3.4.13
+|=======================
+
+Although Kafka 2.0.0 and 2.1.1 use the same version of Zookeeper, Red Hat recommends that you update your Zookeeper cluster to use the newest Zookeeper binaries before proceeding with the main {ProductName} 1.1.0 upgrade.
+endif::StrimziUpgrades[]
+
+*Message format version*
```

Comment:
Please change this into a header, so that we can link to them.

Reply:
Changed